### DISCUSSION TITLE
Penal Codes

### DISCUSSION THREAD

**Discussion by jpitbull917** [SnailyCAD/snaily-cadv4#1387] [Q&A] [answered]
How do you make penal codes for this CAD? Send me an email at jsamons2003@outlook.com

**Comment by casperiv0** [comment, marked as answer]
There’s a template available here: 
https://cad-docs.caspertheghost.me/docs/templates

**Comment by jpitbull917** [comment]
Thak you so much! These are the codes I was needing as well.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
